Assets/Scripts/Models/Furniture.cs

```diff
@@ -34,7 +34,7 @@ public class Furniture : IXmlSerializable, ISelectable, IContextActionProvider,
     /// These actions are called when Trigger is called. They get passed the furniture
     /// they belong to, plus a deltaTime (which defaults to 0).
     /// </summary>
-    protected EventAction eventActions;
+    public EventAction eventActions;
```

Comment:
This can easily be replaced with an automatic property. Do that.

`public EventAction EventAction { get; set; }`